docs/index.rst

```diff
@@ -505,6 +505,39 @@ Sequences help to create different objects in one call:
             ]
         )->create();
 
+Distribute values over a collection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have a collection of objects that you want to distribute over a collection, you can use the ``distribute()`` method:
```

Comment:
```suggestion
If you have a collection of values that you want to distribute over a collection, you can use the ``distribute()`` method:
```

Just want to make it clear that it's not just for collections of objects - could be scalar values also.

Reply:
I'll add another example with an array of scalars :+1: